channel-sdk/src/main/java/com/zello/channel/sdk/Session.kt

```diff
@@ -322,7 +323,9 @@ class Session internal constructor(
     private fun performDisconnect(): Boolean {
         val transport = transport
         state = SessionState.DISCONNECTED
-        this.transport = null
+		channelFeatures = EnumSet.noneOf(ChannelFeature::class.java)
```

Comment:
There appears to exist a mix of spaces and tabs as whitespaces. We use tabs in the rest of our Java/Kotlin codebase, would you replace the spaces with tabs? I suggest setting up the auto-formatting in AS.

Reply:
Man, I thought I had. I'll try to get it fixed.

Reply:
My editor was already set up to use tabs. Looks like maybe the file had previously had spaces instead?